microsoft-edge/progressive-web-apps/ux.md

```diff
@@ -98,7 +132,17 @@ When a user selects the ellipses (**...**) button in the title bar of a Progress
 
 The **App info** menu contains useful information about the app, such as:
 
-*  The app icon, name, and publisher.
-*  The various app permissions that have been granted.
-*  Privacy information such as the number of cookies used.
-*  Lists of extensions and tools that can be used in the app.
+* The app icon, name, and publisher.
```

Comment:
line 129 change from
When a user selects the ellipses (**...**) button in the title bar of a Progressive Web Apps (PWA),
to
When a user selects the ellipses (**...**) button in the title bar of a Progressive Web App (PWA),